src/inc/corinfo.h

```diff
@@ -1584,6 +1584,11 @@ enum CorInfoTokenKind
 
     // token comes from CEE_CONSTRAINED
     CORINFO_TOKENKIND_Constrained = 0x100 | CORINFO_TOKENKIND_Class,
+
+#if COR_JIT_EE_VERSION > 460
```

Comment:
This `#if COR_JIT_EE_VERSION > 460` ifdef is not strictly necessary because the change is backward compatible - everything would work without it just fine. I have added it since it seems to be the convention.